src/app/hero/GetHero.js

```diff
@@ -0,0 +1,5 @@
+const GetHero = ({
+  heroRepository,
+}) => (heroId) => heroRepository.getOne(heroId);
```

Comment:
is need to map to buildError and buildSuccess,

Reply:
True :thinking: 

Just included in the PR.